packages/discount-code-importer/src/main.js

```diff
@@ -97,6 +96,14 @@ export default class DiscountCodeImport {
     return `code in ("${predicateArray.join('", "')}")`
   }
 
+  static promiseMapSerially(functions: Array<Function>) {
```

Comment:
Made this one to replace bluebird's [mapSeries](http://bluebirdjs.com/docs/api/promise.mapseries.html) in order to execute every promise sequentially.